extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -155,9 +155,15 @@ function _load(displaySet, servicesManager, extensionManager) {
   const dataSources = extensionManager.getDataSources();
   const dataSource = dataSources[0];
 
+  const referencedImages = convertAnnotations(displaySet.instance, displaySetService);
```

Comment:
Maybe this should be called getReferencedImages? Alos, why you need this property?

Reply:
The functino returns all the referencedImages array to not call the extension function _getReferencedImagesList that will break with IDC metadata